src/mds/topology/topology_storge_etcd.cpp

```diff
@@ -31,6 +31,43 @@ namespace curve {
 namespace mds {
 namespace topology {
 
+bool TopologyStorageEtcd::LoadPoolset(
+    std::unordered_map<PoolsetIdType, Poolset> *poolsetMap,
+    PoolsetIdType *maxPoolsetId) {
+    std::vector<std::string> out;
+    poolsetMap->clear();
+    *maxPoolsetId = 0;
+    int errCode = client_->List(
+        POOLSETKEYPREFIX, POOLSETKEYEND, &out);
+    if (errCode == EtcdErrCode::EtcdKeyNotExist) {
+        return true;
+    }
+    if (errCode != EtcdErrCode::EtcdOK) {
+        LOG(ERROR) << "etcd list err:" << errCode;
+        return false;
+    }
+    for (int i = 0; i < out.size(); i++) {
+        Poolset data;
+        errCode = codec_->DecodePoolsetData(out[i], &data);
+        if (!errCode) {
+            LOG(ERROR) << "DecodePoolsetData err";
```

Comment:
Print `errCode` here. `errCode` is `int`, do not use it as `bool` like` if (!errCode) {`

Reply:
This refer to physcialPool and logicalPool codes , you can see LoadPhysicalPool func